docs/use-cases/meeting-minutes/2020-01-16 uc-wg meeting notes.md

```diff
@@ -1,3 +1,6 @@
+---
```

Comment:
Is it actually necessary to create these placeholders for metadata? I assume it'd be fine if none of these lines were present.

ditto for the other 4 files like this in the use-cases folder.

Reply:
this was nothing to do with me!   I don't even think these files are linked anywhere